release/pipline/dags/istio_common_dag.py

```diff
@@ -137,6 +137,7 @@ def GenerateTestArgs(**kwargs):
         'DOCKER_HUB',
         'GCS_BUILD_BUCKET',
         'RELEASE_PROJECT_ID',
+        'ISTIO_REPO',
```

Comment:
Would it make sense to try to derive this setting from using GITHUB_ORG & GITHUB_REPO?

Reply:
The idea was to have this be the repo we pull the scripts and the build from and the other variables for pushing the github release to. This was done so I could make test releases. As the scripts only support releasing to the org and repo named in the green build repo, so this extra flexibilityDo is not needed. 

P.S. Do we have a manifest xml maintained for testing. Would yours qualify?

Reply:
Alright, was just a thought on a possible simplification for the default.  We don't have an xml for testing.  If someone is testing a code change then they're probably using a unique branch name (I don't follow that trend and rotate through and reuse branch names like master2, master3, etc. since I find a digit is easier to track and remember than a string [and I don't have to update my manifest repo for a new name] ).

A possible option is to change create_tag_reference.sh and publish_release.sh to just grep on "/$REPO" rather than the full "$ORG/$REPO" .  This would permit your use case (i.e., test builds of unmodified source) using the greenbuilds repo (assuming your forked repo has the istio commits cited by the manifest).  The main tradeoff is if another "istio" repo appears.  The change of this is low, and it'd probably help to grep on "/$REPO " (note the space before the closing quote) to avoid matching repos that have "istio" as a prefix.

Reply:
Actually more like "/$REPO\" " since the value tends to be quoted in the xml.

Reply:
I simplified it. If you change the scripts I'll put this back in